pyvista/utilities/common.py

```diff
@@ -0,0 +1,46 @@
+"""Common functions."""
+import collections.abc
+from typing import Sequence, Union
+
+import numpy as np
+
+PointLike = Union[Sequence, np.ndarray]
+
+
+def _coerce_points_like_arg(points: PointLike,
+                            copy: bool=True) -> np.ndarray:
+    """Check and coerce arg to (n, 3) np.ndarray.
+
+    Parameters
+    ----------
+    points : Sequence(float) or np.ndarray
+        Argument to coerce into (n, 3) ``np.ndarray``.
+    copy : bool
+        Whether to copy the ``points`` array.  Copying always occurs if ``points``
+        is not ``np.ndarray``.
+
+    Returns
+    -------
+    np.ndarray : size (n, 3) array
```

Comment:
Is this syntax correct? I think we usually only have the type here, with a description on a new line. But I'm not sure.